src/Executables/ConvertComposeTable/ConvertComposeTable.cpp

```diff
@@ -46,17 +47,79 @@ void convert_file(const std::string& compose_directory,
+    bpo::options_description command_line_options(
+        "This executable converts an ASCII formatted CompOSE 3d equation of "
+        "state table into a SpECTRE-formatted HDF5 table. This reduces the "
+        "file size by about a factor of 4. We don't use the CompOSE HDF5 "
+        "tables since that requires an HDF5 that works with Fortran.\n"
+        "Note: support for 1d and 2d tables can be added if the CompOSE ASCII "
+        "reader is generalized to support them.\n\n"
+        "Generating the ASCII table using compose:\n"
+        "1.\n"
+        "Download from: https://compose.obspm.fr/software (there's a "
+        "GitLab link)\n\n"
+        "2.\n"
+        "Build by running 'make' in the directory\n\n"
+        "3.\n"
+        "Download EOS from https://compose.obspm.fr/table We will use "
+        "https://compose.obspm.fr/eos/34 as an example. To download, use wget "
+        "on the link from the 'eos.zip' file.\n\n"
+        "4.\n"
+        "Unzip the eos.zip file. This will create a bunch of files in the "
+        "current directory.\n\n"
+        "5.\n"
+        "Run the 'compose' executable in the directory with all the eos "
+        "files. There are 3 main options and you will run the executable "
+        "3 times. Each main option or 'task' has a bunch of numerical value "
+        "inputs."
+        "\n"
+        "Task 1\n"
+        "How many regular thermodynamic quantities...\n"
+        "7\n"
+        "Please select the indices of the thermodynamic quantities...\n"
+        " Index #           1 ?"
+        "1\n"
+        " Index #           2 ?"
+        "2\n"
+        "The remaining are: 3 4 5 7 12\n"
+        "The following function values and derivatives of the free energy...\n"
+        "1\n"
+        "Please select the indices of the thermodynamic...\n"
+        "1\n"
+        "How many particles do you want to select for the file eos.table?\n"
+        "0\n"
+        "There are average mass, charge and neutron numbers...\n"
+        "0\n"
+        "There are microscopic data available of the following type:...\n"
+        "0\n"
+        "There are error estimates available of the following type:...\n"
+        "0\n\n"
+        "Task 2\n"
+        "Temperature interpolation order:\n"
+        "3\n"
+        "Baryon density interpolation order:\n"
+        "3\n"
+        "Hadronic charge fraction interpolation order:\n"
+        "3\n"
+        "beta-equilibrium\n"
+        "0\n"
+        "entropy per baryon\n"
+        "0\n"
+        "Please select the tabulation scheme for the parameters from\n"
+        "1\n"
+        "Get the lower and upper bounds as well as the grid points from the "
+        "compose website for your EOS. Spacing should be\n"
+        "T: log\n"
+        "n_b: log\n"
+        "Y_q: linear\n\n"
+        "Task 3\n"
+        "This will just run, no options needed, but it can take quite a long "
+        "time.\n\n"
+        "Available options are");
 
```

Comment:
If successful, it will list `file eos.table written`, along with the respective labels [and units] of the columns, for example, `1 temperature T [MeV]`.